CHANGELOG.md

```diff
@@ -15,14 +15,22 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+
 # 5.13.1
 
+## @rjsf/mui
+
+- Removed an unnecessary `Grid` container component in the `ArrayFieldTemplate` component that wrapped the `ArrayFieldItemTemplate`, fixing [#3863](https://github.com/rjsf-team/react-jsonschema-form/issues/3863)
+
+## @rjsf/material-ui
+
+- Removed an unnecessary `Grid` container component in the `ArrayFieldTemplate` component that wrapped the `ArrayFieldItemTemplate`, fixing [#3863](https://github.com/rjsf-team/react-jsonschema-form/issues/3863)
+
 ## @rjsf/utils
 
 - Added `getOptionMatchingSimpleDiscriminator()` function
 - `getMatchingOption` and `getClosestMatchingOption` now bypass `validator.isValid()` calls when simple discriminator is provided, fixing [#3692](https://github.com/rjsf-team/react-jsonschema-form/issues/3692)
 
-
```

Comment:
All changes after this point should be reverted... I'm guessing your editor changed the formatting

Reply:
ah oops! will fix just now 😄